src/Dhall/Parser/Combinators.hs

```diff
@@ -0,0 +1,161 @@
+{-# LANGUAGE CPP                        #-}
+{-# LANGUAGE DeriveDataTypeable         #-}
+{-# LANGUAGE GeneralizedNewtypeDeriving #-}
+{-# LANGUAGE OverloadedStrings          #-}
+{-# LANGUAGE RecordWildCards            #-}
+module Dhall.Parser.Combinators where
```

Comment:
Maybe add a header to each module explaining the scope of what belongs in that module

Reply:
:+1: They all have it, except this one. Adding now.

Reply:
Added a missing one, and refined some others.